### ISSUE TITLE
Can I make a pull request?

### ISSUE THREAD

**Issue by ghost** [apparentlymart/angularjs-viewhead#11]
I've made some changes to your repo to:
1. make it more efficient readable (enforcing best practices, as shown here: https://github.com/johnpapa/angular-styleguide), 
2. allow it to be minified without being broken and 
3. added a minified version (someone asked for one in another issue). 

If I could push my branch for a review that'd be great. Everything works exactly the same but allows for more flexibility now. Just waiting on your permission to push my new branch (pulled from master).

**Comment by apparentlymart** [comment]
Sure, please do!

I don't think generated files belong under version control so I wouldn't include the minified version (making that is the responsibility of the user of this library, so they can bundle it with the rest of their code) but I'd definitely accept the other changes.

**Comment by ghost** [comment]
Ah so changing the structure wouldn't apply then would it?

For example, one of the changes was separating the two directives into
separate files and including them in a single module (the viewhead module).
